packages/swirl-components/src/components/swirl-modal/swirl-modal.css

```diff
@@ -382,7 +382,6 @@
 .modal__content {
   overflow-x: hidden;
   overflow-y: auto;
-  min-height: 100%;
```

Comment:
I'm pretty sure this is needed for some use cases. To be safe, you could reset the min-height to 0 of `.modal__content` only within the `.modal--has-header-tools` variant.

Reply:
I added the height property back, does that fix those issues?

Reply:
Not sure. Would have to test the edge cases. Is there anything to be considered against overwriting the min-height for the `modal--has-header-tools` variant only? I think that would be the safest approach with the least risk for regressions.

Reply:
It would work but it feels inconsistent to me. Do you remember what the edge cases were?

Reply:
Nope. We can try without the min-height. Maybe you can make a quick smoke test with some Flip Web and Admin Console modals, once you've updated the Swirl version.